### DISCUSSION TITLE
Better cookie management

### DISCUSSION THREAD

**Discussion by poweruser82** [ViennaRSS/vienna-rss#429] [Ideas]
It would be nice if Vienna had its own cookie jar so it doesn't use Safari's one.
This way you can see pages without having Google, Twitter, Facebooc & co. associate your browsing to the account you're logged in in Safari.

**Comment by josh64x2** [comment]
https://stackoverflow.com/questions/364219/how-can-i-have-multiple-instances-of-webkit-without-sharing-cookies might help us implement this.

**Comment by Eitot** [comment]
WKWebView solves this problem.

**Comment by barijaona** [comment]
It is already solved by recent versions of OS X (Apple changed the behavior).

So, now, Vienna is lacking a cookie manager… 😸

**Comment by poweruser82** [comment]
I'm still on Yosemite. I hope I can update soon

**Comment by jazzace** [comment]
I am looking for the equivalent of Safari's ability to dump existing cookies (Safari > Preferences > Privacy > Manage Website Data or Safari > Clear History). Even better, the ideal solution for me would be the option found in Firefox and OmniWeb to delete cookies every time you quit the app.

**Comment by barijaona** [comment]
Might be of interest for further work on this subject : [BinaryCookieReader](https://github.com/as0ler/BinaryCookieReader).

Usage : `python BinaryCookieReader.py  ~/Library/Cookies/uk.co.opencommunity.vienna2.binarycookies`

**Comment by jazzace** [comment]
This is probably a bigger issue now based on the proliferation of trackers. I know I generally open links in an external browser now due to this shortcoming. It should go back to at least the Nice to Have list.
